frontends/mit-learn/src/pages/HomePage/HeroSearch.tsx

```diff
@@ -185,7 +193,7 @@ const HeroSearch: React.FC = () => {
         </Typography>
         <ControlsContainer>
           <SearchInput
-            placeholder="Search for courses, programs, learning, and teaching materials"
+            placeholder="Search for courses, programs, and learning materials..."
```

Comment:
For a minute i was confused why the `...` was always showing and thought `Input.tsx` had a bug 😁